### PR TITLE
Allow config of file name delimiter.

### PR DESCRIPTION
**Pull request by bemurphy**
Some brunch skeletons appear to be migrating to - instead of _
for filename delimiting.  Supports either via config.

### CODE REVIEW

File: autoload/brunch.vim [hunk 1]
```diff
@@ -54,21 +56,21 @@ function! s:FindBrunchTest(...)
...
     let path = expand('%')
     let path = s:sub(path, g:brunch_path_app, g:brunch_path_test)
-    let path = s:sub(path, '\.', '_test\.') 
+    let path = s:sub(path, '\.', s:name_delim . 'test\.')
     return path
   else
     " name was given, try to find a corresponding file
     let name = a:1
     " controllers are easy to resolve
     if name =~ '_controller'
```
Comment (drichard) [inline autoload/brunch.vim]: Shouldn't this also consider `-controller`?

File: autoload/brunch.vim [hunk 2]
```diff
@@ -54,21 +56,21 @@ function! s:FindBrunchTest(...)
...
-      let path = g:brunch_path_test . '/controllers/' . name . '_test' . s:ext_script
+      let path = g:brunch_path_test . '/controllers/' . name . s:name_delim . 'test' . s:ext_script
     " and so are views
     elseif name =~ '_view'
```
Comment (drichard) [inline autoload/brunch.vim]: Same as above.

**Comment by drichard** [comment]
This should probably use the delimiter as well: https://github.com/bemurphy/vim-brunch/blob/05e1ecbd7bdd0799b4472fe1e247c2a8d206d99d/autoload/brunch.vim#L165

**Comment by bemurphy** [comment]
Sorry, it looks like I missed a bunch of underscores, more than you pointed out even.  I'll go back when I get a sec and track them all down then rewrite my repo.  I'll also go back and test the autocomplete functionality before proceeding

**Comment by drichard** [comment]
Thanks for your work to contribute back to the plugin! I appreciate that.

**Comment by bemurphy** [comment]
Force pushed, think I got them all this time.  I also tested the autocompletes (didn't know they were there, awesome feature!)  Also tried a BStest command from view to test file, worked.

**Comment by drichard** [comment]
Looking good, thanks @bemurphy!

**Pull request merged**